### PR TITLE
windows_defender.xml

### PR DESCRIPTION
**Pull request by Tobias-B-Besemer**
v1.5.0

### CODE REVIEW

File: cleaners/windows_defender.xml [hunk 1]
```diff
@@ -12,20 +12,55 @@
...
+    @note 
 
 -->
 <cleaner id="windows_defender" os="windows">
   <label>Windows Defender</label>
   <description>Anti-virus</description>
   <option id="history">
     <label>History</label>
-    <description>Delete the files</description>
+    <description>Delete the history</description>
```
Comment (az0) [inline cleaners/windows_defender.xml]: "Delete the history" isn't translated, while "Delete the files" is. The label already clarifies the files are history, so is this a worthwhile change?

File: cleaners/windows_defender.xml [hunk 2]
```diff
@@ -12,20 +12,55 @@
...
+    <!-- Definition Update: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\mpas-fe_bd.exe"/>
+    <!-- AS Engine: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Engine.exe"/>
+    <!-- AS Engine Patch: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Engine_Patch_*.exe"/>
+    <!-- AS Base: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Base.exe"/>
+    <!-- AS Base: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Base_Patch*.exe"/>
```
Comment (az0) [inline cleaners/windows_defender.xml]: When using a wildcard, use `search="glob"`

File: cleaners/windows_defender.xml [hunk 3]
```diff
@@ -12,20 +12,55 @@
...
+    <!-- Definition Update: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\mpas-d.exe"/>
+    <!-- Definition Update: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\mpas-fe.exe"/>
+    <!-- Definition Update: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\mpas-fe_bd.exe"/>
+    <!-- AS Engine: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Engine.exe"/>
+    <!-- AS Engine Patch: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Engine_Patch_*.exe"/>
```
Comment (az0) [inline cleaners/windows_defender.xml]: When using a wildcard, use `search="glob"`

File: cleaners/windows_defender.xml [hunk 4]
```diff
@@ -12,20 +12,55 @@
...
+    <!-- AS Base: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Base.exe"/>
+    <!-- AS Base: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Base_Patch*.exe"/>
+    <!-- AntiMaleware Definition Update: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Delta.exe"/>
+    <!-- AntiMaleware Definition Update: -->
+    <action command="delete" search="file" path="%windir%\SoftwareDistribution\Download\Install\AS_Delta_Patch_*.exe"/>
```
Comment (az0) [inline cleaners/windows_defender.xml]: When using a wildcard, use `search="glob"`

**Comment by az0** [comment]
Target to milestone 2.4 for testing

File: cleaners/windows_defender.xml (hunk 1 above)
Comment (Tobias-B-Besemer) [reply cleaners/windows_defender.xml]: Sorry, but it's translated!
2019-03-19 05_12_41-BleachBit

File: cleaners/windows_defender.xml (hunk 2 above)
Comment (Tobias-B-Besemer) [reply cleaners/windows_defender.xml]: Did it in the last version...

File: cleaners/windows_defender.xml (hunk 3 above)
Comment (Tobias-B-Besemer) [reply cleaners/windows_defender.xml]: Done...

File: cleaners/windows_defender.xml (hunk 4 above)
Comment (Tobias-B-Besemer) [reply cleaners/windows_defender.xml]: Yes.

File: cleaners/windows_defender.xml (hunk 1 above)
Comment (az0) [reply cleaners/windows_defender.xml]: I searched for that German string in `de.po`, and it has the extra word "usage"

```
msgid "Delete the usage history"!
msgstr "Den Verwendungsverlauf löschen"!
```

Comment (az0) [reply cleaners/windows_defender.xml]: Resolved in 10467a7aba6b8d921ff553bb96a7f9ae56eb7dd9

**Pull request merged**